server/storage/sql_models.go

```diff
@@ -1,6 +1,15 @@
 package storage
 
-import "github.com/jinzhu/gorm"
+import (
+	"time"
+
+	"github.com/jinzhu/gorm"
+)
+
+const (
+	changeCategoryUpdate   = "update"
```

Comment:
Non-blocking:  Can be done in the future when we actually have a consumer for this as a library, but should these be a type and exported, so someone using the API can check one against the other?